angular-ngrx-scss/src/app/state/profile/profile.state.ts

```diff
@@ -76,6 +76,8 @@ export interface UserReposState {
     url: string;
     node_id: string;
   } | null;
+  fork?: boolean;
+  archived?: boolean;
```

Comment:
question: Are these ever set to undefined or null as the result of any API calls?

Reply:
removed the `?` operator, should always had the value for fork/archived.